apis/cc-en_GB/atrium-api-cc-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/cc/en_GB/mapAssertions.kt

```diff
@@ -80,6 +80,15 @@ inline fun <K, reified V : Any, T: Map<K, V?>> Assert<T>.containsNullable(keyVal
 fun <K> Assert<Map<K, *>>.containsKey(key: K)
     = addAssertion(AssertImpl.map.containsKey(this, key))
 
+/**
+ * Makes the assertion that [AssertionPlant.subject] not contains the given [key].
```

Comment:
Makes the assertion that [AssertionPlant.subject] does not contain the given [key].